packages/experiments/src/components/UnifiedPicker/UnifiedPeoplePicker/examples/DoubleUnifiedPeoplePicker.Example.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import {
+  IFloatingSuggestionItemProps,
+  IFloatingSuggestionItem,
+  IFloatingPeopleSuggestionsProps,
+} from '@uifabric/experiments/lib/FloatingPeopleSuggestionsComposite';
+import { UnifiedPeoplePicker } from '@uifabric/experiments/lib/UnifiedPeoplePicker';
+import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+import { mru, people } from '@uifabric/example-data';
+import { ISelectedPeopleListProps } from '@uifabric/experiments/lib/SelectedItemsList';
+import { IInputProps } from 'office-ui-fabric-react';
+
+const _suggestions = [
```

Comment:
we can skip all this duplication of unifiedpeoplepickerexample code if we just import that into this file ..like this
import { UnifiedPeoplePickerExample } from './UnifiedPeoplePicker.Example';

we can then simply use like you have done
export const DoubleUnifiedPeoplePickerExample = (): JSX.Element => {
  return (
    <>
      To: <UnifiedPeoplePickerExample />
      CC: <UnifiedPeoplePickerExample />
    </>
  );
}